features/support/env.rb

```diff
@@ -2,6 +2,8 @@
 
 # See https://github.com/cucumber/cucumber/wiki/Using-MiniTest
 require 'minitest/spec'
+require 'byebug'
```

Comment:
Debugger should not be loaded by default.

Reply:
✅